### PR TITLE
Language reworked

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

File: docs/admin/access.rst [hunk 1]
```diff
@@ -5,56 +5,56 @@ Access control
...
 Adding permissions to languages, projects or component sets
 +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 
-You can additionally grant permissions to some user based on project, language
-or a component set. To achieve this, create new group (eg. `Czech
-translators`) and configure it for given object. Any assigned permissions will
-be granted to members of that group on selected objects.
+You can additionally grant permissions to any user based on project, language
+or a component set. To achieve this, create a new group (e.g. `Czech
+translators`) and configure it for a given resource. Any assigned permissions will
```
Comment (comradekingu) [inline docs/admin/access.rst]: configure it for a given resource?

File: docs/admin/access.rst [hunk 2]
```diff
@@ -192,12 +192,12 @@ Managing per project access control
 
     This feature only works for ACL controlled projects, see :ref:`acl`.
 
-Users with :guilabel:`Can manage ACL rules for a project` privilege (see
+Users with the :guilabel:`Can manage ACL rules for a project` privilege (see
 :ref:`privileges`) can also manage users in projects with access control
-enabled on the project page. You can add or remove users to the project or make
-them owners.
+turned on through the project page. You can add users, or remove them from a projec,t or make
```
Comment (comradekingu) [inline docs/admin/access.rst]: What does access control turned on through the project page mean?
It is turned on for that, or there?

File: docs/admin/access.rst [hunk 3]
```diff
@@ -5,56 +5,56 @@ Access control
...
-    is built specifically for Weblate. If you are using older version, please
-    consult documentation for that version, information here will not apply.
+    Before Weblate 3.0, the privilege system was based on Django, but is now
+    specifically built for Weblate. If you are using an older version, please
+    the consult documentation for that version, the information here will not apply.
 
-Weblate comes with a fine grained privileges system to assign users permissions
-globally or in limited scope.
+Weblate comes with a fine grained privilege system to assign user permissions
+for the whole instance, or in a limited scope.
```
Comment (comradekingu) [inline docs/admin/access.rst]: Not quite sure what all of the "scope" relates to.

File: docs/admin/access.rst [hunk 4]
```diff
@@ -314,31 +314,31 @@ Screenshots
     Delete screenshot [`Administration`, `Manage screenshots`]
 
 Source strings
-    Edit info on source strings [`Administration`, `Edit source`]
+    Edit source string info [`Administration`, `Edit source`]
```
Comment (comradekingu) [inline docs/admin/access.rst]: Why is this called source strings?

File: docs/admin/access.rst (hunk 3 above)
Comment (nijel) [reply docs/admin/access.rst]: Probably privileges per project, component or language.

File: docs/admin/access.rst (hunk 2 above)
Comment (nijel) [reply docs/admin/access.rst]: It used to be just toggle in some older releases, now it's the Public/Protected/Private choice.

**Comment by nijel** [comment]
Merged, thanks for your contribution!

**Pull request merged**